analyzers/Directory.Build.props

```diff
@@ -6,6 +6,15 @@
              If a setting should only apply to certain projects then add it at a
              lower level.
    -->
+   
+  <PropertyGroup>
+    <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```

Comment:
Don't we need to add <RestoreLockedMode>true</RestoreLockedMode>as well?

Reply:
`RestoreLockedMode` is used during the build as an option: See here: https://github.com/SonarSource/sonar-dotnet/blob/master/azure-pipelines.yml#L42